tuxemon/states/control/__init__.py

```diff
@@ -165,8 +165,7 @@ def change_state(
             label = f"{T.translate(key).upper()}"
             image = self.shadow_text(label)
             item = MenuItem(image, label, None, change_state("SetKeyState", input=input))
-            if input is None:
-                item.enabled = False
+            item.enabled = item is not None
```

Comment:
the name "input" is reserved for python, and its bad practice to "shadow" an existing global or builtin name ("input").  i would say that replaceing "input", with either "button", or "button_name" on line 164 would be best here.
```python
item.enabled = button is not None
```
(change "button" to whatever you rename "input")